contrib/new_post.py

```diff
@@ -54,6 +54,15 @@ class PullRequest():
     labels: list[str]
     user: str
 
+def git_config_get(option, default=None):
+    '''
+    Get named configuration option from git repository.
+    '''
+    try:
+        return subprocess.check_output(['git','config','--get',option]).rstrip().decode('utf-8')
```

Comment:
`check_output()` is the old high-level API: https://docs.python.org/3/library/subprocess.html#older-high-level-api , and most new code uses `subprocess.run()`. Any particular reason to go for `check_output()` here?

Reply:
I ripped it from another script. Agree `run` seems preferred, changed.